ccd-definition/ComplexTypes/LitigationFriend.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "ID": "LitigationFriend",
+    "ListElementCode": "required",
+    "FieldType": "YesOrNo",
+    "ElementLabel": "Is the claimant a child?",
```

Comment:
assuming at a later date that if the applicant were a vulnerable person that this would be part of the question

Reply:
That’s a q for design and business

Reply:
The reason for asking is that if this is to just infer whether the litigant is a child then that should be a flag against the litigant. If its purpose is to infer that a litigant is required then the question should reflect that.